### ISSUE TITLE
can you make a feature that tell if a story whatsapp has been deleted or not

### ISSUE THREAD

**Issue by Shizu-ka** [tomer8007/whatsapp-web-incognito#109]
can you make a feature that tell if a story whatsapp has been deleted or not. sometimes it makes a problem because i reply to a story which someone has deleted

**Comment by IsaacHatton** [comment]
When a status is downloaded (with debug mode) this is outputted to console:
![image](https://github.com/tomer8007/whatsapp-web-incognito/assets/74068072/81b40cd7-055c-4b96-aaec-473abe6938a3)
Could there be a way to store these, then reference them when a status is opened to check if there is a match, then create a popup?

**Comment by tomer8007** [comment]
Yes, knowing which status was revoked is the easy part. The more annoying part in my opinion is to identify the status ID from the DOM (probably with `FindReact`) and then change the UI.
